### PR TITLE
[chore] Adds an SDK language and version to the heads passed to cdp-service

### PR DESCRIPTION
**Pull request by deangalvin-cb**
### What changed? Why?

This will help telemetry so we can support customers & identify bad versions better.

#### Qualified Impact

During the auth header addition we will also append sdk version & language information

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: pkg/auth/transport.go [hunk 1]
```diff
@@ -31,6 +31,12 @@ func (t *transport) RoundTrip(req *http.Request) (*http.Response, error) {
 	}
 
 	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", jwt))
+	fmt.Sprintf(
+		"%s,%s",
+		fmt.Sprintf("%s=%s", "sdk_version", "0.0.6"),
+		fmt.Sprintf("%s=%s", "sdk_language", "go"),
+	)
+	req.Header.Set("Correlation-Context", fmt.Sprintf("Bearer %s", jwt))
```
Comment (alex-stone) [inline pkg/auth/transport.go:39]: Should this be:
```suggestion

	req.Header.Set(
		"Correlation-Context", 
		fmt.Sprintf(
			"%s,%s",
			fmt.Sprintf("%s=%s", "sdk_version", "0.0.6"),
			fmt.Sprintf("%s=%s", "sdk_language", "go"),
		),
	)
```

Comment (alex-stone) [reply pkg/auth/transport.go:39]: Do we have a way of dynamically seeting the `sdk_version`?

Comment (deangalvin-cb) [reply pkg/auth/transport.go]: Not really, I looked into ways to pull it, but its a lot of go "debug" packages, or oddities. I think once we have a solid `make version` type of script we can just append it there, but until then we can do this scrappy way with the PR comment!

**Review by alex-stone — APPROVED** [review]

**Pull request merged**